pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -432,13 +433,64 @@ func newReduceV1Hint(x hinter.ResOperander) hinter.Hinter {
 	}
 }
 
-func createReduceV1Hinter(resolver hintReferenceResolver) (hinter.Hinter, error) {
+func createReduceHinter(resolver hintReferenceResolver) (hinter.Hinter, error) {
+	x, err := resolver.GetResOperander("x")
+	if err != nil {
+		return nil, err
+	}
+
+	return newReduceHint(x), nil
+}
+
+// ReduceEd25519 hint reduces a packed value modulo the Curve25519 prime
+//
+// `newReduceEd25519Hint` takes 1 operander as argument
+//   - `x` is the packed value to be reduced
+//
+// `newReduceEd25519Hint` assigns the result as `value` in the current scope
+func newReduceEd25519Hint(x hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "ReduceEd25519",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack (V1) //> from starkware.cairo.common.cairo_secp.secp_utils import pack (V2)
```

Comment:
format line?

Reply:
oh yeah i made a mistake it was the wrong comment, just pushed an update !